keel/api/v1/auth/views.py

```diff
@@ -1325,8 +1325,25 @@ def upload(self, request, format='json'):
         user_id = user.id
         files = request.FILES
 
-        doc_type = req_data.get("doc_type")
+        case_id = req_data.get("case","")
+
+        # Catching the case Id exception unless FE start passing it, TODO: Remove it once done
+        try:
+            from keel.api.v1.cases.serializers import CaseIDSerializer
 
+            # validate Case ID against User/Agent
+            case_serializer = CaseIDSerializer(data = {"case_id": case_id, "user_id": user_id})
+            case_serializer.is_valid(raise_exception=True)
+            case_obj = case_serializer.validated_data
+            
+            # actual user containing the document
+            doc_user_id = case_obj.user_id
+
+        except:
+            doc_user_id = user_id
+            pass
```

Comment:
we can use log here...

Reply:
The try except is added only to make the API backward compatible, since before `case_id` was not expected for this API.  but once FE start sending the case_id, will remove this try except. 

Do we still need the log?

Reply:
ok,,,got it!